app.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+"""
+Usage:
+    amity (-i | --interactive)
+    amity (-h | --help | --version)
+    amity create_room <room_name> <room_type>
+    amity add_person <person_name> <person_type> <wants_space>
+    amity reallocatePerson <person_name> <room_name>
+    amity print_allocations [--o=filename]
+    amity print_unallocated [--o=filename]
+    amity load_people [--o=flename]
+
+
+Options:
+    -i, --interactive  Interactive Mode
+    -h, --help  Show this screen and exit.
+    --baud=<n>  Baudrate [default: 9600]
+"""
+
+import sys
+import cmd
+import os
+from docopt import docopt, DocoptExit
+from dbmodels import create_db
+from models.Amity import Amity
+from sessions import Sessions
+
+
+def docopt_cmd(func):
+    """
+    This decorator is used to simplify the try/except block and pass the result
+    of the docopt parsing to the called action.
+    """
+
+    def fn(self, arg):
+        try:
+            opt = docopt(fn.__doc__, arg)
+
+        except DocoptExit as e:
+            # The DocoptExit is thrown when the args do not match.
+            # We print a message to the user and the usage block.
+
+            print('Invalid Command!')
+            print(e)
+            return
+
+        except SystemExit:
+            # The SystemExit exception prints the usage for --help
+            # We do not need to do the print here.
+
+            return
+
+        return func(self, opt)
+
+    fn.__name__ = func.__name__
+    fn.__doc__ = func.__doc__
+    fn.__dict__.update(func.__dict__)
+    return fn
+
+
+class Amity (cmd.Cmd):
+    intro = 'Welcome to my Amity allocation!' \
+        + ' (type help for a list of commands.)'
+    prompt = '<amity>'
+    amity = Amity()
```

Comment:
You've named your class 'Amity' and also are importing a class called 'Amity'. So from which Class is this instance of Amity from?